policy-controller/src/admission.rs

```diff
@@ -98,13 +93,14 @@ impl hyper::service::Service<Request<Body>> for Service {
     }
 }
 
-fn json_response(rsp: AdmissionReview) -> Result<Response<Body>, anyhow::Error> {
-    let bytes = serde_json::to_vec(&rsp)?;
-    Response::builder()
+fn json_response(rsp: AdmissionReview) -> Result<Response<Body>> {
+    let bytes = serde_json::to_vec(&rsp)
+        .map_err(|e| anyhow::Error::from(e).context("failed to encode admission review as JSON"))?;
```

Comment:
eh, this should probably be
```suggestion
        .with_context(|| "failed to encode admission review as JSON")?;
```